src/headers.ts

```diff
@@ -115,3 +123,12 @@ export function withHeaders(
 
   return new Response(inputResponse.body, responseInit);
 }
+
+/**
+ * Whether the client supports Brotli compression.
+ */
+export function supportsBrotli(
```

Comment:
One more thing: We should return `vary: accept-encoding` to all requests.

Reply:
Should this be added to the SHARED_HEADERS map?

Reply:
Yah

Reply:
Done a4ba1ed